playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -90,6 +90,10 @@
     - install:code
 
 - include: customer_theme.yml
+  tags:
+    - install
+    - install:code
+    - update_lms_theme
```

Comment:
DRY fix - just put the tags once on the include statement and they will be evaluated for each task in the playbook.